src/Core/Localization/Specification/Factory.php

```diff
@@ -100,7 +100,8 @@ public function buildPriceSpecification(
         CldrLocaleInterface $cldrLocale,
         Currency $currency,
         $numberGroupingUsed,
-        $currencyDisplayType
+        $currencyDisplayType,
```

Comment:
Rather than using an optional argument, maybe you can force it?
Although this represent a BC break this factory shouldn't be extended anyway and thus it should have be `final` as well

Reply:
No, I can't force it. By default, it used the MaxFractionDigits from the currency of the CldrLocale object. But I need specifically to overwrite this default value.